src/main/java/io/cdap/plugin/cloud/vision/action/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Actions to build CDAP pipelines that use the Cloud Vision API.
+ */
+package io.cdap.plugin.cloud.vision.action;
```

Comment:
generally don't need pacakage-info files Can you remove them?

Reply:
removed